vuepress/docs/next/docs/reference/bundle-upgrade.md

```diff
@@ -0,0 +1,136 @@
+---
+sidebarDepth: 2
+---
+
+# Bundles Upgrade Guideline
+
+Two types of bundles can be utilized on Entando 7.1+. The original git-based bundle added a composable method with reusable components. The newly introduced docker-based bundles optimize their structure and expands functionality. The [Bundle Evolution Comparison](./bundle-comparison.md) table lists the differences between the two types. The following page describes the guidelines for upgrading to docker-based v5 bundles.
+
+Git-based bundles do not need to be to upgraded. Both types of bundles are compatible on Entando 7.1+, but docker-based bundles provide several innovations and an improved developer experience.
+
+The improvements include:
+* Simplified bundle directory structure
+* A decoupled build and deployment process that is easier to understand, modify, and troubleshoot
+* CI/CD integration and support 
+* One Git source repository required with Docker image repositories created automatically as needed
+* Optimized file storage and transfer methods using a layered file system, beneficial for large projects
+* Standardized bundle address format
+
+## Quick Reference
+* [Bundle Specifications](../curate/bundle-details.md)
+
+* [Bundle Management](../getting-started/ent-bundle.md)
+
+* [Bundle Version Comparison Table](./bundle-comparison.md)
+
+## Upgrade Process
+
+### Initialize Bundle 
+
+1. Initialize a new bundle to set up the standard Entando Bundle structure using the ent CLI:
+
+``` 
+ent bundle init YOUR-BUNDLE-NAME
+```
+### Copy Assets
+1. Use the ent CLI bundler command to export assets from an existing Entando Bundle. See the step-by-step [Export and Publish Tutorial](../../tutorials/create/pb/export-bundle-from-application.md). 
+
+2. `YOUR-BUNDLE-NAME/platform` directory is designated for resources such as fragments, contents, pages, and templates. Copy the output of the ent bundler to this folder. You can also accomplish this manually but there are advantages to the bundler such as generating top level descriptors for platform resources.
+Note that widgets that are not MFEs are considered platform entities on Entando and should be placed in the `platform/widgets` directory.
+
+### Add Micro Frontends & Microservices
+ 
+1. Add your micro frontend to the new bundle descriptor `entando.json`:
+``` sh
+ent bundle mfe add YOUR-MFE1
+```
+The docker-based bundle has a centralized bundle descriptor with bundle entities such as micro frontends, microservices, APIs, DBMS, and ID management specifications. 
```

Comment:
this can act as a >Note: (grayed out aside)

"Docker-based bundles have" - no "the" adding specificity; this is a general statement that applies to all docker-based bundles.

**with** -> **that includes** ("with" is alongside while "that includes" = contains)